Application/Business/LevelServices/LevelServices.cs

```diff
@@ -21,6 +22,19 @@ public async Task<LevelEntity> GetById(Guid levelId)
         return await _levelRepo.FoundOrThrowAsync(levelId,
             Constants.ENTITY.LEVEL + Constants.ERROR.NOT_EXIST_ERROR);
     }
+    public async Task<ICollection<LevelEntity>> List(Guid[] levelIds)
+    {
+        List<LevelEntity> result = new List<LevelEntity>();
+        foreach (var levelId in levelIds.Distinct())
+        {
+            var level = await _levelRepo.FoundOrThrowAsync(levelId, "Level with ID " + levelId + " " + Constants.ERROR.NOT_EXIST_ERROR);
```

Comment:
I think we don;t need throw errow here, let use where async to load the level in list